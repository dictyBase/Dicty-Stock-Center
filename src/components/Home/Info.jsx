// @flow
import React from "react"
import LinkList from "../LinkList"
import { PanelBlue } from "styles"

const info = [
  { name: "Order Information", to: "/information/order", routerAware: true },
  {
    name: "Payment Information",
    to: "/information/payment",
    routerAware: true,
  },
  {
    name: "Deposit Information",
    to: "/information/deposit",
    routerAware: true,
  },
]

/**
 * Displays all of the info page links
 */

const Info = () => {
  return (
    <PanelBlue>
      <LinkList list={info} />
    </PanelBlue>
  )
}

export default Info
