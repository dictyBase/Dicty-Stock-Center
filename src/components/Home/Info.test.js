import React from "react"
import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import "jest-styled-components"
import Info from "./Info"

test("matching a snapshot of Info", () => {
  const component = renderer.create(
    <BrowserRouter>
      <Info />
    </BrowserRouter>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
