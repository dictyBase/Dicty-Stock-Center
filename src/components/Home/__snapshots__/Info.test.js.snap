// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of Info 1`] = `
<div
  className="sc-cSHVUG dyYMvt"
>
  <div
    className="sc-kAzzGY FynjO"
  >
    <ul>
      <li>
        <a
          href="/orders/information"
          onClick={[Function]}
        >
          Order Information
        </a>
      </li>
      <li>
        <a
          href="/payments/information"
          onClick={[Function]}
        >
          Payment Information
        </a>
      </li>
      <li>
        <a
          href="/deposits/information"
          onClick={[Function]}
        >
          Deposit Information
        </a>
      </li>
    </ul>
  </div>
</div>
`;
